server/game/GameActions/PlaceFateAction.js

```diff
@@ -6,19 +6,27 @@ class PlaceFateAction extends CardGameAction {
     setDefaultProperties() {
         this.amount = 1;
         this.origin = null;
+        this.recipient = null;
     }
 
     setup() {
         this.name = 'placeFate';
-        this.targetType = [CardTypes.Character];
+        this.targetType = [CardTypes.Character, 'player'];
         this.effectMsg = 'place ' + this.amount + ' fate on {0}';
     }
 
     canAffect(card, context) {
-        if(this.amount === 0 || card.location !== Locations.PlayArea) {
+        return super.canAffect(card, context) && this.checkOrigin(context) && this.checkRecipient(card, context);
+    }
+
+    checkRecipient(card, context) {
+        if(!this.recipient || ['player', 'ring'].includes(this.recipient.type)) {
+            return true;
+        }
+        if(this.amount === 0 || (card && card.location !== Locations.PlayArea)) {
             return false;
         }
-        return super.canAffect(card, context) && this.checkOrigin(context);
+        return this.recipient.allowGameAction('placeFate', context);
```

Comment:
I'm not really sure what the changes to `canAffect` and adding `checkRecipient` are intended to achieve...